lib/land/config.rb

```diff
@@ -75,5 +75,9 @@ def new_visit_reasons=(value)
       raise ArgumentError, "must be a subset of #{ALLOWED_NEW_VISIT_REASONS}" unless (value - ALLOWED_NEW_VISIT_REASONS).empty?
       @new_visit_reasons = value
     end
+
+    def logger
+      @logger ||= (defined?(Rails) ? Rails.logger : Logger.new($stdout))
+    end
```

Comment:
`Land.config.logger.error "Land::Trackers::ApiTracker Error: #{error}"` was returning the error `undefined method 'logger' for an instance of Land::Config (NoMethodError)`. So adding this allows `Land.config.logger` to be defined and accessible to use from the `api_tracker.rb`.